### ISSUE TITLE
speed up code

### ISSUE THREAD

**Issue by lindamegner** [innosat-mats/MATS-L1-processing#54]
Right now flatfield and dark current images are binned for every image. This should be done only once for each setting and channel.

**Comment by OleMartinChristensen** [comment]
status of this?

**Comment by lindamegner** [comment]
Still needs to be done. We discussed this. We need to create a lookup table similar to what you did for the linearity stuff and save each binned dark and flatfield, and then we should check int the code if the a binned version exists.

**Comment by OleMartinChristensen** [comment]
ok, I remove the "prior to molflow tag then" and we fix it later

**Comment by skymandr** [comment]
This is done, right? @OleMartinChristensen

**Comment by OleMartinChristensen** [comment]
I dont know @lindamegner

**Comment by lindamegner** [comment]
No, this is still done in the time consuming way of re-binning the flatfield/dark current field for every image. So it should be changed. I am availabe mornings this week (until 10 generally although only to 9 today) if you are around and want to discuss it.
